### ISSUE TITLE
traceObserver setter is missing

### ISSUE THREAD

**Issue by scanline** [ruffle-rs/ruffle#11406]
### Describe the bug

While trying to find a way get the `trace()` messages of a movie running inside ruffle, I stumbled upon the [traceObserver](https://ruffle.rs/js-docs/master/classes/RufflePlayer.html#traceObserver) accessor which should let me assign a callback function to receive the actual trace output.
Unfortunately the setter function seems to be missing from ruffle.
So something like e.g.:
```
<html lang="en">
<head>
	<meta charset="utf-8">
	<script src="https://unpkg.com/@ruffle-rs/ruffle"></script>
</head>
<body>
	<script type="text/javascript">
		window.RufflePlayer = window.RufflePlayer || {};

		const ruffle = window.RufflePlayer.newest();
		const player = ruffle.createPlayer();
		console.log(player.readyState);
		console.log(player.metadata);
		console.log(player.isPlaying);
		console.log(player.volume);
		console.log(player.fullscreenEnabled);
		console.log(player.isFullscreen);
		console.log(player.traceObserver);
	</script>
</body>
</html>
```

returns **undefined** for `console.log(player.traceObserver);`

### Expected behavior

Something like:
```
window.RufflePlayer = window.RufflePlayer || {};
const ruffle = window.RufflePlayer.newest();
const player = ruffle.createPlayer();

player.traceObserver((message) => {
	console.log(message);
})
```

should print the `trace()` messages from the running .swf movie to the browser's console.

### Affected platform

Self-hosted version

### Operating system

Windows 10

**Comment by n0samu** [comment]
You don't need to do anything so complicated. Just set Ruffle's `logLevel` to "info". See https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#configuration-options

**Comment by scanline** [comment]
Thanks @n0samu - that indeed works. I just can see it as a workaround though as the traceObserver setter should do what it's supposed to do. ;)

**Comment by n0samu** [comment]
The traceObserver setter actually works fine, but you can only use it after Ruffle has finished loading (the `player.load` promise resolves). See the code snippet at https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#advanced-usage
The reason `player.traceObserver` always returns `undefined` is that it actually only has a setter defined, not a getter. Which is definitely silly, but doesn't prevent you from accomplishing your goal here.

**Comment by scanline** [comment]
Well, `console.log(player.traceObserver)` should log the setter function - which it does not because it is not present.
I already tried what you suggested because I also had the suspicion the movie might have to finish loading first but that does not work either.
So something like
```
player.load("test.swf").then(() => {

	player.traceObserver((msg) => {
		console.log("msg", msg);
	})
	console.info("Ruffle successfully loaded the file");
});
```

throws:
**Uncaught (in promise) TypeError: player.traceObserver is not a function**

**Comment by n0samu** [comment]
No, traceObserver is a property, not a function. It should be `player.traceObserver = console.log`
